### ISSUE TITLE
Bombardments damage is inaccurate

### ISSUE THREAD

**Issue by Saviorself300** [Tercioo/Details-Damage-Meter#837]
Bombardments damage for Scalecommander Devastation Evoker is not being tracked correctly. Details would show only ~3 million damage done, and Warcraft Logs would show ~40 million damage done.

**Comment by Flamanis** [comment]
It is being tracked correctly. Bombardment triggers damage for the other players. We are not given the events that wcl and other out of game logging tools are that say what damage was contributed by another player.

**Comment by Flamanis** [comment]
Yes and no. A plugin would not be able to do that exactly. But you could have a custom display that shows a bar for that

**Comment by Justeroo** [comment]
Would the damage events be duplicated if that was done? I have no idea how to make or do this, just wondering about some of the ins and outs. Say you had a custom display doing this, would you still have bomb damage showing up for your raid members, then a separate bar showing all bomb damage?

**Comment by Flamanis** [comment]
You could filter out bombardment damahe from others. It would take some extra processing, but it would be purely visual as opposed to what we did for kyrian and such which was creating a fake actor object that held all of the damage info

**Comment by Saviorself300** [comment]
Would is be possible to have a lighter bar next to the damage bar that estimates the bombardment damage, similar to what Aug has?

**Comment by Flamanis** [comment]
Closing this issue as it is old and could have already been fixed. If you believe this to be in error, or if the issue still persists, please create a new issue.
